tests/test_astropy.py

```diff
@@ -45,8 +45,23 @@ def test_angle():
 def test_basic():
     """Basic tests of CelestialCoord construction and corresponding SkyCoord construction.
     """
-    pass
 
+    t0 = time.time()
+    c1 = coord.CelestialCoord(0.234 * coord.radians, 0.342 * coord.radians)
+    t1 = time.time()
+
+    a1 = astropy.coordinates.SkyCoord(0.234, 0.342, unit=units.radian,
+                                      frame=astropy.coordinates.FK5(equinox='J2000'))
+    t2 = time.time()
+
+    np.testing.assert_allclose(c1.rad, [a1.ra.rad, a1.dec.rad], rtol=1.e-5,
+                               err_msg='starting coords in coord/astropy different')
```

Comment:
I know you copied this from test_precess, but "starting" seems inappropriate here, since there isn't any further action on the coordinate.  Probably want to change what the error message is in this case.